Makefile

```diff
@@ -38,6 +38,9 @@ endif
 -include current_platform.mk
 include $(CRAZYFLIE_BASE)/tools/make/platform.mk
 
+# Create a define for the platform (upper case), for instance PLATFORM_CF2
+CFLAGS += -DPLATFORM_$(shell echo $(PLATFORM) | tr a-z A-Z)=1
```

Comment:
nit: this could be:

```
CFLAGS += -DPLATFORM_$(shell echo $(PLATFORM) | tr '[:lower:]' '[:upper:]')=1
```

Not sure if it is better though, a bit more self-explanatory.